internal/services/datafactory/data_factory_trigger_schedule_data_source.go

```diff
@@ -0,0 +1,318 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package datafactory
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/datafactory/mgmt/2018-06-01/datafactory" // nolint: staticcheck
+	"github.com/hashicorp/go-azure-sdk/resource-manager/datafactory/2018-06-01/factories"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/datafactory/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/datafactory/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type TriggerScheduleDataSource struct{}
+
+type TriggerScheduleDataSourceModel struct {
+	Name          string                    `tfschema:"name"`
+	DataFactoryID string                    `tfschema:"data_factory_id"`
+	Description   string                    `tfschema:"description"`
+	Schedule      []TriggerScheduleSchedule `tfschema:"schedule"`
```

Comment:
I'm thinking this should be 
```suggestion
	Schedule      []TriggerSchedule `tfschema:"schedule"`
```

Reply:
Yeah, well, it's less than ideal naming on my part, but since it's a schedule of a "trigger schedule" resource... ;) I'll rename.